### ISSUE TITLE
Change dropdown genreList to checkboxes for multiple selection

### ISSUE THREAD

**Issue by somanytextbooks** [somanytextbooks/AndrewProject#2]
Change dropdown genreList to checkboxes for multiple selection
This needs to be done for both index.html and createMovie.html
and their corresponding php files searchMovie.php and createMovie.php

**Comment by somanytextbooks** [comment]
Add for loop to allow search for multiple genres to turn up items that match all criteria, then provide partial matches below

**Comment by somanytextbooks** [comment]
Add logic so that second list can exist minus the genre selected in the first list (ie you cant search for action action)
Also, second pop down should be generated at start or only after first one is chosen to make this easier?

**Comment by somanytextbooks** [comment]
AddMultipleSelectionGenre - Branch
Commit #1
Added checkboxes to createMovie.
Also added logic loadGenre.php to distinguish whether it needed to pass
a dropdown list for searching (index.html) or checkboxes for creating
(createMovie.html)  index.html doesnt seem to be respecting ifelse
statement though...

**Comment by somanytextbooks** [comment]
Logic is working, mistake not reloading page properly. index.html does respect if statement.  Error prints weird though, in hidden spot.  May be something that needs to be changed.

Todo:
Get php to accept multiple genre values in query
Second dropdown for searchpage

**Comment by somanytextbooks** [comment]
jQuery use $('#id :checked').each function(){ to pull data from checkboxes.  Will then put in an array to pass

**Comment by somanytextbooks** [comment]
Commit 16bc60dac33081f4477cb9623033ef1667878661:

**Comment by somanytextbooks** [comment]
Perhaps a better logic structure for sql is to have search

SELECT * FROM movie_genre_view WHERE (genreID=? AND movie LIKE ?) AND (genreID=? AND movie LIKE ?)

**Comment by somanytextbooks** [comment]
Completed:
Pass php multiple genre values (as an array) - Done
Get php to accept multiple genre values in query - Done
 - Done
To do:
SQL statement to search for both genres
      Create php foreach loop to deal with these values in order of relevance (genre1 & 2, then genre1 then genre2)
Second dropdown for searchpage - animate to hide when genre1 not set

**Comment by somanytextbooks** [comment]
Commit 07f9982:

Animated second dropdown list for genre2

To do:
SQL statement to search for both genres
Create php foreach loop to deal with these values in order of relevance (genre1 & 2, then genre1 then genre2)
createMovie.php - make accept multiple genres
createMovie.html - pass multiple values

**Comment by somanytextbooks** [comment]
Will not pass to php if no genres selected

**Comment by somanytextbooks** [comment]
createMovie.php check that not passed 2 same genre values (if (var1!==var2) {} else { ERROR})

**Comment by somanytextbooks** [comment]
Commit: 5cca694
Merged WIP Broken into AddMultipleSelectionGenre since I'm happy with where its going.
Added SQL statement for multigenres, doesnt work completely

Currently this will return Die Hard Comedy, Die Hard Action when
searching Action, Comedy for genres and no title.  Where are just
actions and just comedies?  Also add CONCAT so that Die Hard is just one
printout with Action/Comedy being its genre field.

**Comment by somanytextbooks** [comment]
Commit 55d2915
Dropdown list 2 now does not include what was selected in list 1
Could be more efficient though, since calls php to repopulate the list each time - created new issue for this to polish

**Comment by somanytextbooks** [comment]
Check to make sure that a movie doesnt already exist when creating!

**Comment by somanytextbooks** [comment]
Commit 7af56b6
Changed sql tables to add movie ID to movie_genre_view.  Renamed some columns too so will have to get the php pages to reflect these changes

**Comment by somanytextbooks** [comment]
Made the table printed by searchMovie.php grab and give table headings from SQL query
Commit: ad48fe8
